src/Domain/CampaignService.php

```diff
@@ -35,6 +35,44 @@ public function __construct(
     ) {
     }
 
+    /**
+     * @return array<string, mixed>
+     */
+    public function renderCampaignSummary(CampaignDomainModel $campaign): array
+    {
+        $sfCampaignData = $campaign->getSalesforceData();
+
+        $campaignId = $campaign->getId();
+
```

Comment:
will probably want to convert this to use a serializable class soon, but literal array works as long as we don't need to use the same structure twice. I think it's going to need to be a class as soon as we implement search which will return the same structure.

Reply:
Agreed we'll want this soon, and that it doesn't block doing it this way for now.